gensim/test/test_matutils.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Licensed under the GNU LGPL v2.1 - http://www.gnu.org/licenses/lgpl.html
+
+"""
+Automated tests for checking various math helper functions.
+"""
+
+
+import logging
+import unittest
+
+import numpy as np
+
+from gensim import matutils
+
+
+class TestMatUtils(unittest.TestCase):
+    def test_dirichlet_expectation_keeps_precision(self):
```

Comment:
I don't think that make new file is a good idea. Please move this tests to LDA tests class.

Reply:
Also, please add tests for the load **old** model with **new code** for all models that you changed.

Reply:
Test not just "load", but also using the loaded models.

The asserts that we newly sprinkled into the code may trigger errors in various places, if something is wrong.

Reply:
@menshikh-iv That test don't involve LDA at all, so it's wrong place for this test. Haven't found any file involving testing matutils functions so I think a separate file is not that bad.